python/ray/tests/test_mini.py

```diff
@@ -16,28 +16,32 @@ def f_simple():
         return 1
 
     assert ray.get(f_simple.remote()) == 1
+    assert ray.get(f_simple.remote()) == 1
+    assert ray.get(f_simple.remote()) == 1
 
-    # Test multiple return values.
-
-    @ray.remote(num_return_vals=3)
-    def f_multiple_returns():
-        return 1, 2, 3
-
-    x_id1, x_id2, x_id3 = f_multiple_returns.remote()
-    assert ray.get([x_id1, x_id2, x_id3]) == [1, 2, 3]
-
-    # Test arguments passed by value.
-
-    @ray.remote
-    def f_args_by_value(x):
-        return x
-
-    for arg in test_values:
-        assert ray.get(f_args_by_value.remote(arg)) == arg
-
-    # Test arguments passed by ID.
 
-    # Test keyword arguments.
+#
+#    # Test multiple return values.
+#
+#    @ray.remote(num_return_vals=3)
+#    def f_multiple_returns():
+#        return 1, 2, 3
+#
+#    x_id1, x_id2, x_id3 = f_multiple_returns.remote()
+#    assert ray.get([x_id1, x_id2, x_id3]) == [1, 2, 3]
+#
+#    # Test arguments passed by value.
+#
+#    @ray.remote
+#    def f_args_by_value(x):
+#        return x
+#
+#    for arg in test_values:
+#        assert ray.get(f_args_by_value.remote(arg)) == arg
+#
+#    # Test arguments passed by ID.
+#
+#    # Test keyword arguments.
```

Comment:
Revert these changes?

Reply:
Sorry, don't remember to modify this file.

Reply:
Can you git checkout this file from master to undo the changes?

Reply:
done